src/deadline/nuke_submitter/deadline_submitter_for_nuke.py

```diff
@@ -239,6 +285,12 @@ def _get_parameter_values(
     parameter_values.append(
         {"name": "ProxyMode", "value": "true" if settings.is_proxy_mode else "false"}
     )
+
+    # Set the OCIO config path value
+    if nuke_ocio.is_OCIO_enabled():
+        ocio_config_path = get_ocio_config_path()
+        parameter_values.append({"name": "OCIOConfigPath", "value": ocio_config_path})
```

Comment:
Should we check if the path is `None`?

Reply:
Although it should not happen when OCIO is enabled, I added safe guard for this just in case, somehow it happens, customers would know what they should do to unblock themselves. Thanks!